patroni/postgresql/citus.py

```diff
@@ -0,0 +1,386 @@
+import logging
+import re
+import time
+
+from six.moves.urllib_parse import urlparse
+from threading import Condition, Event, Thread
+
+from .connection import Connection
+from ..dcs import CITUS_COORDINATOR_GROUP_ID
+from ..psycopg import connect, quote_ident
+
+CITUS_SLOT_NAME_RE = re.compile(r'^citus_shard_(move|split)_slot(_[1-9][0-9]*){3}$')
```

Comment:
The third number is only added in the current main branch of citus. To be compatible with citus 11.1, it should also match slots with 2 numbers.
```suggestion
CITUS_SLOT_NAME_RE = re.compile(r'^citus_shard_(move|split)_slot(_[1-9][0-9]*){2,3}$')
```